frontend/src/components/NotReadyProjectWarning.vue

```diff
@@ -0,0 +1,50 @@
+<!--
+SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+<template>
+  <v-tooltip top v-if="projectDetails.isNotReady">
+    <template v-slot:activator="{ on }">
+      <v-icon v-if="projectDetails.phase === 'Terminating'" :small="small" v-on="on" color="primary" class="ml-1">mdi-delete-sweep</v-icon>
+      <v-icon v-else-if="projectDetails.phase === 'Initial'" :small="small" v-on="on" color="primary" class="ml-1">mdi-plus-circle-outline</v-icon>
+      <v-icon v-else :small="small" v-on="on" color="warning" class="ml-1">mdi-alert-circle-outline</v-icon>
+    </template>
+    <div>
+      The project phase is <v-chip color="primary" label x-small>{{projectDetails.phase}}</v-chip>
```

Comment:
```suggestion
      The project phase is <v-chip color="primary" label x-small class="px-1">{{projectDetails.phase}}</v-chip>
```

old:


<img width="491" alt="Screenshot 2022-07-26 at 19 22 28" src="https://user-images.githubusercontent.com/5526658/181071113-ede905e9-3ec8-4740-a3b2-41fcce2a2eef.png">

new:
<img width="482" alt="Screenshot 2022-07-26 at 19 22 14" src="https://user-images.githubusercontent.com/5526658/181071066-dc171560-6b58-476c-acb7-8cdd1b1e37eb.png">

and maybe we can highlight / make the header bold?

Reply:
The lower smaller text